### ISSUE TITLE
No more events received after a while

### ISSUE THREAD

**Issue by ab-taktik** [CCob/bittrex4j#16]
Hello,

Thanks for the great library. 
I want to endlessly listen to exchange deltas and put fills in ElasticSearch. It works well for a few minutes/hours, then suddently no more messages are received.

See graph below : 

<img width="989" alt="screen shot 2018-02-18 at 10 46 14" src="https://user-images.githubusercontent.com/5154448/36350517-47453a9c-1499-11e8-9a7f-689b353f134d.png">

A bit after 6AM I stop receiving messages. There are no errors in the logs. 

Do you have an idea on what could be the cause of this? Maybe I should re-subscribe every X minutes by calling subscribeToExchangeDeltas? But I saw that the "marketSubscriptions" variable in your code is a list, not a set, so I guess you don't expect multiple subscriptions for the same market.

Maybe there is some sort of keepAlive to send? 

Thanks for your help, 

Antoine

**Comment by ab-taktik** [comment]
The last logs are : 

```
18-02-2018 14:16:07.750 [pool-2-thread-1] INFO  c.g.c.b.BittrexExchange.WebSockets.log - HubConnection - Slow connection detected
18-02-2018 14:16:14.415 [pool-2-thread-1] INFO  c.g.c.b.BittrexExchange.WebSockets.log - HubConnection - Timeout
18-02-2018 14:16:14.422 [pool-2-thread-1] INFO  c.g.c.b.BittrexExchange.WebSockets.log - HubConnection - Restarting the transport
18-02-2018 14:16:14.422 [pool-2-thread-1] INFO  c.g.c.b.BittrexExchange.WebSockets.log - HubConnection - Starting the transport
```

But then nothing happens

**Comment by CCob** [comment]
Hi ab-taktik.  Give 1.0.6-SNAPSHOT a try.  The Java version of signalr is poor at best, especially reconnect behavior.  I have updated the forked version of signalr which fixes some of this logic and the lastest bittrex4j SNAPSHOT uses this update.  Would love some feedback on this.  Been using it myself the last week and the reconnect logic seems to work successfully.  You will need to handle the connected handler to register for updates once again though as the whole socket is torn down and CloudFlare is also performed again on reconnect

**Comment by ab-taktik** [comment]
Thanks! I'll give it a try, I'll keep you posted

**Comment by ab-taktik** [comment]
It's much better indeed! has been running for 3 days now without interruption

**Comment by CCob** [comment]
Excellent, glad to hear it.  I'll close this ticket then.  The changes will be in the 1.0.6 release which I'll be doing soon

**Comment by CCob** [comment]
Fixed in 1.0.6 Release
